ui/component/privacyAgreement/view.jsx

```diff
@@ -0,0 +1,197 @@
+            )}
+          </h3>
+        </div>
+        <div>
+          <img src={image} />
+        </div>
+      </div>
+      <Form onSubmit={handleSubmit} className="section__body">
+        <p className="section__subtitle">
+          {__('Can this app send information about your usage to inform publishers and improve the software?')}
+        </p>
+        <fieldset>
+          <FormField
+            name={'shareFreely'}
+            type="radio"
+            label={
+              <>
+                {/*<span className="emoji">😄</span>*/}
+                {__('Yes, including with third-party analytics platforms')}
+              </>
+            }
+            helper={
+              <I18nMessage
+                tokens={{
+                  terms: (
+                    <Button
+                      tabIndex="2"
+                      button="link"
+                      href="https://www.lbry.com/termsofservice"
+                      label={__('Terms of Service')}
+                    />
+                  ),
+                }}
+              >
+                Sending information to third parties (e.g. Google Analytics or Mixpanel) allows us to use detailed
+                analytical reports to improve all aspects of LBRY.
+              </I18nMessage>
+            }
+            checked={share === 'free'}
+            onChange={e => setShare('free')}
+          />
+          <FormField
+            name={'shareWithLBRY'}
+            type="radio"
+            checked={share === 'limited'}
+            label={
+              <>
+                {/*<span className="emoji">🙂</span>*/}
+                {__('Yes, but only with LBRY, Inc.')}
+              </>
+            }
+            helper={
+              <I18nMessage
+                tokens={{
+                  terms: (
+                    <Button
+                      tabIndex="2"
+                      button="link"
+                      href="https://www.lbry.com/termsofservice"
+                      label={__('Terms of Service')}
+                    />
+                  ),
+                }}
+              >
+                Sharing information with LBRY, Inc. allows us to report to publishers how their content is doing, as
+                well as track basic usage and performance. This is the minimum required to earn rewards from LBRY, Inc.
+              </I18nMessage>
+            }
+            onChange={e => setShare('limited')}
+          />
+          <FormField
+            name={'shareNot'}
+            type="radio"
+            checked={share === 'none'}
+            label={
+              <>
+                {/*<span className="emoji">😢</span>*/}
+                {__('No')}
+              </>
+            }
+            helper={
+              <I18nMessage
+                tokens={{
+                  terms: (
+                    <Button
+                      tabIndex="2"
```

Comment:
I don't think these should all have the same `tabIndex`? That could mess with tabbing through with your keyboard